tests/coverage_lcov/read.me

```diff
@@ -0,0 +1,42 @@
+# The LLVM-COV -- is a tool shows code coverage information for programs that are instrumented to emit profile data.
```

Comment:
Please do not insert lcov doc here. Simply provide a link to it.

Reply:
Okay.